test/quantization/fx/test_quantize_fx.py

```diff
@@ -3199,6 +3199,38 @@ def forward(self, x):
         # checking result match
         self.assertEqual(out_ref, out)
 
+    def test_conv_lowering(self):
+        convs = [nn.Conv1d, nn.Conv2d, nn.Conv3d]
```

Comment:
nit: we typically use a map from dim to conv class, e.g.: https://github.com/pytorch/pytorch/blob/master/test/quantization/fx/test_quantize_fx.py#L3619, that might be clearer